cfme/base/ui.py

```diff
@@ -40,6 +40,14 @@ def address(self):
     return self.appliance.url
 
 
+def prepare_server_name(name):
+    # there is a bug in CF 5.10 UI when it has different number of spaces in different html attrs
+    # this is temporary workaround until this BZ is fixed
+    new_name = re.escape(name)
```

Comment:
BZ please, at least in comment, maybe even use BZ().blocks to control flow here so we automatically stop working around it?

Reply:
Of course. I haven't submitted it yet.  I was working on workaround.